assemble/pom.xml

```diff
@@ -92,6 +92,12 @@
       <artifactId>protobuf-java</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+      <version>2.3.0.1</version>
```

Comment:
This version can be omitted if the dependency is listed in the root `pom.xml`'s `<dependencyManagement/>` section.

Reply:
Thanks @ctubbsii for your comments and suggestions, indeed helpful. I was checking the failure from the logs and I could co-relate the pom conventions. generally I follow https://accumulo.apache.org/contributor/building and run mvn verify -Psunny. For this run probably focus was more on testing it manually after deployment. I am correcting the fix and will be pushing the change with detailed comments.

Reply:
And yes, had manually tested with Uno deployment.

Output from /rest/xml as below.(snip)
`<?xml version="1.0" encoding="UTF-8" standalone="yes"?><stats><servers><server id="man-dev:9997"><hostname>man-dev:9997</hostname><lastContact>6211</lastContact><responseTime>0</responseTime><osload>0.37</osload><version>2.1.0-SNAPSHOT</version><compactions.........</stats>`

output from /rest/json as below. (snip)
`{"servers":[{"server":{"hostname":"man-dev:9997","lastContact":7436,"responseTime":2,"osload":0.02,"version":"2.1.0-SNAPSHOT","compactions".....`